src/integration.jl

```diff
@@ -168,3 +168,47 @@ function tight_remainder(f, tm::T, xm::T, x0::Interval, max_steps::Integer=20) w
     end
     return xOld
 end
+
+
+
+function integrate(f::TaylorModelN, which=1, x0=0)
+
+    p = integrate(f.pol, which)  # not necessary if an already complete Taylor series, in which case p2 == f.p
+
```

Comment:
I think you can add to `p[0]` the initial condition `x0`, directly after `p` is defined, instead of doing it after `g` is created.